### PR TITLE
Add a section about when/how to write I18N Considerations

### PR DESCRIPTION
**Pull request by aphillips**
Addresses w3c/i18n-actions#19

Adds a subsection to the Introduction dealing with how and when to write an internationalization considerations section. The specific details need to be reviewed by the WG.

***
<a href="https://pr-preview.s3.amazonaws.com/aphillips/bp-i18n-specdev/pull/112.html" title="Last updated on Jul 27, 2023, 3:27 PM UTC (560af40)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/bp-i18n-specdev/112/8037702...aphillips:560af40.html" title="Last updated on Jul 27, 2023, 3:27 PM UTC (560af40)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -167,6 +167,32 @@ <h3>Create a github checklist</h3>
...
+<h3>When and how to write an <em>Internationalization Considerations</em> section in your spec</h3>
+
+<p class="advisement">All additions of or changes to an <cite>Internationalization Considerations</cite> section MUST be reviewed by the Internationalization (i18n) WG.</p>
+
+<p>Specifications are not required to include a special section or appendix describing internationalization considerations of their specification. In general, the Internationalization (i18n) WG prefers that information about language, regional, or cultural variation, support, or adaptation appear in the body of the specification, closely associated with the relevant features.</p>
+
+<p>However, there are a few cases in which you should consider providing a section like this. You should consider including an internationalization considerations section when:
+<ul>
+	<li>International features require additional explanation that would otherwise interfere with or clutter-up the body of the specification.</li>
+	<li>You wish to provide examples of features, such as localization, without interfering with the body of the specification.</li>
```
Comment (r12a) [inline index.html]: Perhaps add?
> For example, summaries of the general approach taken, or factors that affect the approach taken throughout the document.

File: index.html [hunk 2]
```diff
@@ -167,6 +167,32 @@ <h3>Create a github checklist</h3>
...
+</ul></p>
+
+<p>If you decide to create an internationalization considerations section, you should do the following:</p>
+<ol>
+	<li>Raise an issue in your specification's github repo with the title <cite>Internationalization Considerations</cite> and apply the label <code>i18n-needs-resolution</code>. This will alert the I18N WG of your intention to write a section that requires review. (This also automatically creates a tracking issue in the I18N WG's repo.)</li>
+	<li>Mention the issue in any pull request that adds or modifies the section.</li>
+	<li>Mention the issue in your horizontal review request to the I18N WG.</li>
+</ol>
+
+<p>Your internationalization considerations section should have the title <cite>Internationalization Considerations</cite> or <cite>Internationalization (I18N) Considerations</cite>. It normally should appear as an appendix in your specification. The order relative to other appendices is up to you.</p>
```
Comment (r12a) [inline index.html]: I18N should probably be I18n or i18n.

**Review by r12a — APPROVED** [review]
Just a few light editorial comments to consider. Otherwise, looks quite good!

I'm always missing an option to finish my review that says "Approve (but check out my comments first)", so i'll click on Approve rather than Request Changes.

File: index.html (hunk 1 above)
Comment (aphillips) [reply index.html]: Done

File: index.html (hunk 2 above)
Comment (aphillips) [reply index.html]: Done.

**Review by r12a — APPROVED** [review]

**Pull request merged**